routeros/provider_schema_helpers.go

```diff
@@ -170,6 +170,12 @@ var (
 		Description: "Layer2 Maximum transmission unit. " +
 			"[See](https://wiki.mikrotik.com/wiki/Maximum_Transmission_Unit_on_RouterBoards).",
 	}
+	PropL2MtuRw = &schema.Schema{
```

Comment:
in ethernet devices l2mtu is writable (and must be written).

Reply:
I suggest not putting this attribute in the helpers, as it only occurs in this form (RW) in one resource so far.

Reply:
KeyL2Mtu: {
                Type:     schema.TypeInt,
		Optional: true,
		Description: "Layer2 Maximum transmission unit. " +
			"[See](https://wiki.mikrotik.com/wiki/Maximum_Transmission_Unit_on_RouterBoards).",
	}